library/src/c_api.rs

```diff
@@ -51,8 +51,11 @@ fn app_config_from_c(c_params: *const AppParameters) -> updater::AppConfig {
 
     updater::AppConfig {
         cache_dir: to_rust(c_params_ref.cache_dir),
-        version_name: to_rust(c_params_ref.version_name),
-        version_code: c_params_ref.version_code,
+        release_version: format!(
```

Comment:
@eseidel is this the correct/idiomatic way of doing this?

Reply:
Yup.  format!() is the way I know.  It produces a String (an owned character buffer) which is what you want.